bayesian/api_v1.py

```diff
@@ -286,6 +306,11 @@ def get(ecosystem, package, version):
             _session.post(url=METRICS_SERVICE_URL + "/api/v1/prometheus", json=metrics_payload)
 
             return result
+        elif os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW", "") == "1":
+            msg = f"No data found for {ecosystem} package {package}/{version} " \
+                   "ingetion flow skipped as DISABLE_UNKNOWN_PACKAGE_FLOW is enabled"
+
+            raise HTTPError(202, msg)
```

Comment:
how is 202 an error?

Reply:
It was returning 404 as first proposal, there was review comment to have 202 and do nothing as we have disabled unknown package flow.

Reply:
It was returning 404 as first proposal, there was review comment to have 202 and do nothing as we have disabled unknown package flow.

Reply:
202 Accepted
The request has been accepted for processing, but the processing has not been completed. The request might or might not be eventually acted upon, and may be disallowed when processing occurs.

how can it be error?

Reply:
@arajkumar @prashbnair why cant we simply return a valid response with 202.. why are we raising an error with 202? this doesnt sound right.